catroid/src/org/catrobat/catroid/ui/MarketingActivity.java

```diff
@@ -84,6 +84,9 @@ public void onClick(View v) {
 			@Override
 			public void onClick(View v) {
 				String url = ProjectManager.getInstance().getCurrentProject().getXmlHeader().getUrl();
+				if (!url.trim().startsWith("http://")) {
+					url = "https://share.catrob.at" + url;
```

Comment:
you should always use constants instead of hardcoded values (see Constants.java).